lib/Controller/ViewController.php

```diff
@@ -100,6 +100,7 @@ public function index():TemplateResponse {
 		$showWeekNumbers = $this->config->getUserValue($this->userId, $this->appName, 'showWeekNr', $defaultWeekNumbers) === 'yes';
 		$skipPopover = $this->config->getUserValue($this->userId, $this->appName, 'skipPopover', $defaultSkipPopover) === 'yes';
 		$timezone = $this->config->getUserValue($this->userId, $this->appName, 'timezone', $defaultTimezone);
+		$attachmentsFolder = $this->config->getUserValue($this->userId, $this->appName, 'attachmentsFolder', '/');
```

Comment:
Default to `/calendar`, not the root folder?

Reply:
I did so because the /Calendar folder is missing by default. And deleted the code that creates the /calendar folder before uploading local attachments. any ideas how to create folder once?

Reply:
Checking for the folder each time involves a network request, too. I suggest checking the folder once on opening the calendar app and then store the boolean. The MKCOL operation can be done once an attachment file needs to be created if the folder does not exist yet.

Reply:
Hello! So, you can test this. I updated the code.